content/news/this-week-in-redox-35.md

```diff
@@ -0,0 +1,213 @@
++++
+title = "This Week in Redox 35"
+author = "goyox86"
+date = "2018-01-26 15:54:24 +0000"
++++
+
+This is the 35th post of a series of blog posts tracking the development and progress of Redox, the Rust operating system. If you want to know more about Redox in general, visit our [Github page](https://github.com/redox-os/redox).
+
+*(edited by [@goyox86](https://github.com/goyox86))*
+
+# PSA
+
+If you have any questions, ideas, or are curious about Redox, we recommend joining `#redox` on `irc.mozilla.org`, [our Discourse forum](https://discourse.redox-os.org/) or you can get an invite to our chat by sending an email request to `info@redox-os.org`.
+
+# What's new in Redox?
+
+## TL;DR
+
+Hello people! Welcome to the 35th edition of TWiRx!
+
+We started this year with [@jackpot51](https://github.com/jackpot51) being featured on the [Changelog](https://changelog.com/podcast/280) talking about building a secure operating system in Rust and about Redox in general. Make sure you listen to the episode, it's pretty good! (specially the part on which Jeremy says **SPOILER ALERT!**: "Redox is not going away")
+
+This month we've also got the [Redox Crash Challenge](https://www.reddit.com/r/rust/comments/7rv9vw/redox_os_crash_challenge/) going on. An initiative to attempt to cause the system to:
```

Comment:
It is worth noting that a number of the issues found have been fixed: 

- An issue was fixed with too many arguments to exec not returning E2BIG: https://github.com/redox-os/kernel/issues/81
- An issue was fixed with the kernel attempting to remap kernel code (and failing due to protections) in: https://github.com/redox-os/kernel/issues/79
- An issue was fixed with address validation allowing overflows: https://github.com/redox-os/kernel/commit/dcb49be48133cf811dcd870e8af5f7ffaffc2ab0
- An issue with `su` allowing login as any user by pressing <kbd>Ctrl</kbd>+<kbd>D</kbd>: https://github.com/redox-os/userutils/commit/02759b4a5a347726e6e81d4ee46a2ade86fd9e1e